packages/page-supersig/src/Extrinsics/Submission.tsx

```diff
@@ -0,0 +1,140 @@
+// Copyright 2017-2022 @polkadot/app-extrinsics authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { SubmittableExtrinsic, SubmittableExtrinsicFunction } from '@polkadot/api/types';
+import type { RawParam } from '@polkadot/react-params/types';
+import type { DecodedExtrinsic } from './types';
+import type { ApiPromise } from '@polkadot/api';
+import type { Call } from '@polkadot/types/interfaces';
+import React, { useCallback, useState } from 'react';
+import { Button, Extrinsic, InputAddress, MarkError, TxButton } from '@polkadot/react-components';
+import { useApi } from '@polkadot/react-hooks';
+import { BalanceFree } from '@polkadot/react-query';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from './translate';
+import { assert, isHex } from '@polkadot/util';
+import Decoded from './Decoded';
+
+interface Props {
+  className?: string;
+  defaultValue?: DecodedExtrinsic | null;
+  setLast?: (value: DecodedExtrinsic | null) => void;
+}
+interface DefaultExtrinsic {
+  defaultArgs?: RawParam[];
+  defaultFn: SubmittableExtrinsicFunction<'promise'>;
+}
+
+function extractDefaults (value: DecodedExtrinsic | null, defaultFn: SubmittableExtrinsicFunction<'promise'>): DefaultExtrinsic {
+  if (!value) {
+    return { defaultFn };
+  }
+  
+  return {
+    defaultArgs: value.call.args.map((value) => ({
+      isValid: true,
+      value
+    })),
+    defaultFn: value.fn
+  };
+}
+
+function getDecodedExtr (api:ApiPromise, encodedVal: string, defaultValue: DecodedExtrinsic | null): DecodedExtrinsic|null {
+  try {
+    if(encodedVal == undefined && defaultValue){
+      return defaultValue;
+    }
+    let extrinsicCall: Call;
+    let hex = encodedVal;
+    let decoded: SubmittableExtrinsic<'promise'> | null = null;
+    assert(isHex(hex), 'Expected a hex-encoded call');
+
+    try {
+      decoded = api.tx(hex);
+      extrinsicCall = api.createType('Call', decoded.method);
+    } catch (e) {
+      extrinsicCall = api.createType('Call', hex);
+    }
+    const { method, section } = api.registry.findMetaCall(extrinsicCall.callIndex);
+    const extrinsicFn = api.tx[section][method];
+
+    return { call: extrinsicCall, fn: extrinsicFn, hex};
+  } catch (e) {
+    return null;
+  }
+}
+
+function Selection ({ className, defaultValue }: Props): React.ReactElement<Props> {

```

Comment:
This is extrinsics duplicated?

Sadly I believe his is a dead-end for this type of functionality in here. It is completely un-integrated in the UI. For instance with current multisig I can submit a transaction from anywhere in the apps UI and I can sign it and create a multisig operation.

Basically whet this means it is a stand-along, non-integrated into the apps UI section. I cannot for instance 

- point the supersig to a proxy - and do do anything, e.g. transfer, I would need to create the full proxy extrinsic myself
- I cannot just change my identities in accounts and get it signed via the normal process - I would need to get the data somehow and fill it in here
- ...

Reply:
sorry this can also be deleted, again i just brought it locally to see how it worked.